data/alumni.yml

```diff
@@ -232,23 +232,67 @@
     - url: http://www.bouquetapp.com
       icon: link
 
-- name: Emmanuel Louisy-Gabriel
-  image_name: manu.png
-  url: https://github.com/ManyBaby
+- name: Caroline Carrière
+  image_name: caroline.jpg
+  url: https://fr.linkedin.com/pub/caroline-carriere/40/36/b1b
   project:
-    name: We are the Diggers
-    url: http://wearethediggers.herokuapp.com
-    image_name: wearethediggers-landing.jpg
+    name: Ma Share École
+    url: https://www.mashareecole.com/
+    image_name: mashareecole-landing.jpg
     teaser:
-      fr: Une appli web pour identifier des soldats inconnus australiens ayant combattu en France en 14 - 18. A partir d'une collection unique !
-      en: Help us identify unknown Australian soldiers who fought in France during WWI. From of a unique collection of pictures!
+      fr: Marketplace et plateforme de mise en relation des parents dont les enfants vont dans la même école.
+      en: Marketplace and social network for parents whose children are going to the same school.
     description:
-      fr: Développement de l'application web et d'un robot de scraping pour récupérer des données militaires à partir des sites officiels australiens (noms, grades, identifiants des régiments & bataillons..)
-      en: Development of the web-application and of the scraping engine to collect military data from official Australian sites (names, grades, regiment ID, etc)
+      fr: "Beta de l'application Rails et intégration graphique, interface de connexion avec code parent sécurisé."
+      en: Development of Ruby on Rails web-app Ma Share École and setup of a secure connexion access for the parents.
   links:
-    - url: https://github.com/ManyBaby
+    - url: https://fr.linkedin.com/pub/caroline-carriere/40/36/b1b
+      icon: linkedin
+    - url: https://www.facebook.com/pages/MA-SHARE-ÉCOLE/1422248518078090
+      icon: facebook
+    - url: https://www.mashareecole.com/
+      icon: link
+
+- name: Valentin Chaput
+  image_name: valentin.jpg
+  url: https://github.com/valentinchaput
+  project:
+    name: Parol
+    url: http://parol-production.herokuapp.com/
+    image_name: parol-landing.jpg
+    teaser:
+      fr: Le Github de l'Assemblée nationale
+      en: Github for the french law
+    description:
+      fr: "Développement d'un scraper de Légifrance, connexion à l'API GitHub pour le stockage des lois / amendements et prototypage de l'interface de visualisation."
+      en: "Scraping of Légifrance, connexion to the Github API and UX/UI development of the web-app."
```

Comment:
GItHub for the French law. Légifrance scraping, GitHub API integration and UI prototyping of the rails applicaiton.